gdraw/ggdkdraw.c

```diff
@@ -2441,25 +2441,17 @@ GDisplay *_GGDKDraw_CreateDisplay(char *displayname, char *UNUSED(programname))
     gdisp->selinfo[sn_user1].sel_atom = gdk_atom_intern_static_string("PRIMARY");
     gdisp->selinfo[sn_user2].sel_atom = gdk_atom_intern_static_string("PRIMARY");
 
-    bool tbf = false, mxc = false;
-    int user_res = 0;
-    GResStruct res[] = {
-        {.resname = "MultiClickTime", .type = rt_int, .val = &gdisp->bs.double_time},
-        {.resname = "MultiClickWiggle", .type = rt_int, .val = &gdisp->bs.double_wiggle},
-        {.resname = "SelectionNotifyTimeout", .type = rt_int, .val = &gdisp->sel_notify_timeout},
-        {.resname = "TwoButtonFixup", .type = rt_bool, .val = &tbf},
-        {.resname = "MacOSXCmd", .type = rt_bool, .val = &mxc},
-        {.resname = "ScreenResolution", .type = rt_int, .val = &user_res},
-        {.resname = NULL},
-    };
-    GResourceFind(res, NULL);
-    gdisp->twobmouse_win = tbf;
-    gdisp->macosx_cmd = mxc;
+    GDrawResourceFind();
+    gdisp->bs.double_time = _GDraw_res_multiclicktime;
```

Comment:
I think I've expressed this elsewhere, but not really a fan of these random/externed variables for preferences (not saying though that that needs to change for this pr, just longer term goals for how preferences should be managed).

In that sense imo the old approach (or the interface at least) was cleaner/makes more sense to me.

Reply:
Yeah, I suppose I added some more of that but it's still fairly modest.

In this case the "old approach" suffered from code duplication and a lack of an editing interface. It seemed strange to add the whole block to both back-ends. 

It would certainly be easy enough to shove them all into a struct and give it a proper header if you think that would help.